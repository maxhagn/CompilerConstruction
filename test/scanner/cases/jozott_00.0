0
01
$0a
000010
$000a