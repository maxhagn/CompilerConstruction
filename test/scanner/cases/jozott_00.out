num 0
num 1
num 10
num 10
num 10
